sage-ios/SAGE/SAGE/EditProfileController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  EditProfileController.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 1/17/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class EditProfileController: FormController {
+    
+    var user: User
+    
+    init(user: User) {
+        self.user = user
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let editProfileView = EditProfileView(frame: self.view.frame)
```

Comment:
you should just make an editProfileView property and override loadView() which just has the line self.view = self.editProfileView